### PR TITLE
chore: migrate to Command5, Core3

### PR DESCRIPTION
**Pull request by WillieRuemmele**
### What does this PR do?

QA:
this should be tested along-side other commands that interact with the aliases, such as `org:create -a` and others

PARKING ORBIT

upgrades to core3/command5

requires https://github.com/salesforcecli/command/pull/60

mostly done as exploration for the parking-orbit process for other plugins

### What issues does this PR fix or reference?
@W-10743766@

### CODE REVIEW

File: src/commands/alias/set.ts [hunk 1]
```diff
@@ -21,12 +20,14 @@ export default class Set extends AliasCommand {
 
   public async run(): Promise<AliasResult[]> {
     const varargs = this.varargs || {};
-    const valuesToSet = Object.keys(varargs).map((v) => `${v}=${varargs[v] as string}`);
-    const savedValues = (await Aliases.parseAndUpdate(valuesToSet, AliasGroup.ORGS)) as Dictionary<string>;
-    const results = Object.keys(savedValues).map((alias) => ({
-      alias,
-      value: savedValues[alias],
-    }));
+    const info = await GlobalInfo.create();
```
Comment (peternhale) [inline src/commands/alias/set.ts:23]: @WillieRuemmele why are you using `GlobalInfo.create()` vs `GlobalInfo.getInstance()`?

File: src/commands/alias/unset.ts [hunk 2]
```diff
@@ -16,24 +16,25 @@ export default class Unset extends AliasCommand {
...
-    const argv = this.parseArgs();
+    const argv = await this.parseArgs();
 
     if (!argv || argv.length === 0) {
-      throw SfdxError.create('@salesforce/plugin-alias', 'unset', 'NoAliasKeysFound', []);
+      throw messages.createError('NoAliasKeysFound');
     } else {
       const results: AliasResult[] = [];
-      const aliases = await Aliases.create(Aliases.getDefaultOptions());
+      const info = await GlobalInfo.create();
```
Comment (peternhale) [inline src/commands/alias/unset.ts:25]: Same here?

**Review by peternhale — CHANGES_REQUESTED** [review]
@WillieRuemmele just a couple of questions

File: src/commands/alias/set.ts (hunk 1 above)
Comment (WillieRuemmele) [reply src/commands/alias/set.ts:23]: probably just an artifact from `find + replace` - updating 👍

**Review by peternhale — APPROVED** [review]

**Comment by peternhale** [comment]
QA:
`sfdx alias list` ✅ 
`sfdx alias set` ✅ 
`sfdx alias unset` ✅

**Comment by peternhale** [comment]
QA

Ran the following and the changes appear to be on par with current version of commands.
``` bash
% ~/plugin-alias/bin/run alias:list
=== Alias List

 Alias      Value                            
 ────────── ──────────────────────────────── 
 xxxxxxxxxx admin@xxxxxxxxxx.org 
% ~/plugin-alias/bin/run alias:set foobarbaz=xxxxxxxxxx
=== Alias Set

 Alias     Value      
 ───────── ────────── 
 foobarbaz xxxxxxxxxx 
% ~/plugin-alias/bin/run alias:list                    
=== Alias List

 Alias      Value                            
 ────────── ──────────────────────────────── 
 xxxxxxxxxx admin@xxxxxxxxxx.org 
 foobarbaz  xxxxxxxxxx                       
% ~/plugin-alias/bin/run alias:list --json
{
  "status": 0,
  "result": [
    {
      "alias": "xxxxxxxxxx",
      "value": "admin@xxxxxxxxxx.org"
    },
    {
      "alias": "foobarbaz",
      "value": "xxxxxxxxxx"
    }
  ]
}
% ~/plugin-alias/bin/run alias:unset foobarbaz
=== Alias Unset

 Alias     Success 
 ───────── ─────── 
 foobarbaz true    
% ~/plugin-alias/bin/run alias:list           
=== Alias List

 Alias      Value                            
 ────────── ──────────────────────────────── 
 xxxxxxxxxx admin@xxxxxxxxxx.org 
% ~/plugin-alias/bin/run alias:set foobarbaz=doesnotexist@all.org
=== Alias Set

 Alias     Value                
 ───────── ──────────────────── 
 foobarbaz doesnotexist@all.org 

 ```

**Pull request merged**